src/ops/nn.jl

```diff
@@ -204,10 +204,23 @@ end
 @not_implemented function all_candidate_sampler()
 end
 
-@not_implemented function atrous_conv2d()
+@not_implemented function atrous_conv2d(value, filters, rate, padding; name="")
```

Comment:
In the new style, a default name should here in the function definition instead of a blank string.